### PR TITLE
adds a pacman transformation reagant

### PR DESCRIPTION
**Pull request by boodaliboo**
a fun little chemical that admins can bus to transform people to pacman which is a lot easier then var editting (contrary to popular belief most admins don't want to var edit just to do a gimmick and will choose to do something easier instead of editing lines of code
#### Changelog

:cl:  
rscadd: Added the best chemical (pacman)
/:cl:

### CODE REVIEW

**Comment by overlord12345** [comment]
epic

**Comment by boodaliboo** [comment]
> epic

games

**Comment by boodaliboo** [comment]
why are you a bully ktl what do you have against pacman

File: code/datums/diseases/transformation.dm [hunk 1]
```diff
@@ -225,6 +225,23 @@
 	new_form = /mob/living/simple_animal/hostile/morph
 	infectable_biotypes = list(MOB_ORGANIC, MOB_INORGANIC, MOB_UNDEAD) //magic!
 
+/datum/disease/transformation/pacman
+	name = "Pac-Man's Curse"
+	cure_text = /datum/reagent/consumable/nothing
```
Comment (Ktlwjec1) [inline code/datums/diseases/transformation.dm]: cure text "nothing"

File: code/datums/diseases/transformation.dm [hunk 2]
```diff
@@ -225,6 +225,23 @@
...
+	cures = list(/datum/reagent/medicine/adminordrazine)
+	agent = "Pac-Man's Curse"
+	desc = "A 'gift' from somewhere amazing"
```
Comment (Ktlwjec1) [inline code/datums/diseases/transformation.dm]: a full stop "A 'gift' from somewhere amazing."

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -630,6 +630,16 @@
 
 /datum/reagent/gluttonytoxin/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
 	L.ForceContractDisease(new /datum/disease/transformation/morph(), FALSE, TRUE)
+	
+/datum/reagent/pacmanimfunnyrightilikepacmanthatmakesmefunnyplease
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: pacmanimfunnyrightilikepacmanthatmakesmefunnyplease change this

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 4]
```diff
@@ -630,6 +630,16 @@
...
+	name = "Pac-Man's Curse"
+	description = "An advanced corruptive toxin produced by something great"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: a full stop "An advanced corruptive toxin produced by something great."

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
Comment (boodaliboo) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: ok

**Comment by boodaliboo** [comment]
did them all

File: code/datums/diseases/transformation.dm [hunk 5]
```diff
@@ -225,6 +225,23 @@
 	new_form = /mob/living/simple_animal/hostile/morph
 	infectable_biotypes = list(MOB_ORGANIC, MOB_INORGANIC, MOB_UNDEAD) //magic!
 
+/datum/disease/transformation/pacman
+	name = "Pac-Man's Curse"
+	cure_text = /datum/reagent/consumable/adminordrazine
```
Comment (Ktlwjec1) [inline code/datums/diseases/transformation.dm]: Adminordrazine

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]

File: code/datums/diseases/transformation.dm (hunk 5 above)
Comment (boodaliboo) [reply code/datums/diseases/transformation.dm]: oh it doesnt use a path im dumb

File: code/datums/diseases/transformation.dm [hunk 6]
```diff
@@ -225,6 +225,23 @@
 	new_form = /mob/living/simple_animal/hostile/morph
 	infectable_biotypes = list(MOB_ORGANIC, MOB_INORGANIC, MOB_UNDEAD) //magic!
 
+/datum/disease/transformation/pacman
+	name = "Pac-Man's Curse"
+	cure_text = adminordrazine
```
Comment (Ktlwjec1) [inline code/datums/diseases/transformation.dm]: cure_text = "adminordrazine"

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]

File: code/datums/diseases/transformation.dm (hunk 6 above)
Comment (boodaliboo) [reply code/datums/diseases/transformation.dm]: wrong fake news

**Comment by alexkar598** [comment]
This was already closed wasn't it

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
Comment (AsV9) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: lmao what the fuck

**Comment by boodaliboo** [comment]
It was but I was suggested to reopen it because your reasoning for closing it was bad

**Comment by Hopekz** [comment]
I liked this PR originally.
Alex's reason for closing it was kind of wonky he said something like "You can already do this with [*insert a full command to type out here*] ) .

Sometimes admins just want to do things with buttons instead of typing a full command.
Making this a reagent will make it more accessible for people that also don't know this exists.

**This PR is a quality of life improvement not everyone is a script kiddie we have some normal people administrating this server.**
+1

**Comment by AsV9** [comment]
@Hopekz You don't need to be a script kiddie to find this button:
image

**Comment by Hopekz** [comment]
> @Hopekz You don't need to be a script kiddie to find this button:

I see this button in VV as I use this menu.
Sorry maybe I wasn't too clear about what value I think this PR would add and reading back my comment I see that I made it seem like there almost was no way to turn someone into pacman without scripting.

This is a fictional example, but imagine some admin is trying to bus in a scavenger hunt with a potion or pill at the end where if ingested transforms an individual into pacman.
Right now this is only possible if you were to be constantly watching your bus "victim".
It is a convenience feature not because it is not accessible but because it is inconvenient to have to babysit someone in this fictional example.

**Do I have to have this option?** No.
**Should I have this option if I choose to use it?** Maybe your choice I don't merge I only give opinions.

**Pull request closed without merging**